data/json/npcs/hints.json

```
@@ -163,7 +163,7 @@
       "A friend of mine was a hunter and showed me once how to butcher properly.  You need a flat clean surface to place the products, a rack to hang the carcass in the air, and a good knife.  If you're in a forest you may use a tree and a rope.  Big game might require a saw too.",
       "A friend of mine was a hunter and told me, that if you field dress a corpse, it will stay fresh a bit longer.",
       "Look at the sky before you go for adventure.  You will know what weather to expect.  It's nothing compared to the old meteorology and satellite pictures, but at least you may know if you need the umbrella.",
-      "Be extra careful on roads.  They are easy to travel on, but occasional minefields or a road block can make you feel sorry in an instant.  I've even seen a tank once.  I ran away like never before.",
+      "Be extra careful on roads.  They are easy to travel on, but occasional road blocks can make you feel sorry in an instant.  I've even seen a tank once.  I ran away like never before.",
```

There are no tank bots either.
```suggestion
      "Be extra careful on roads.  They are easy to travel on, but occasional road blocks can make you feel sorry in an instant.",
```